### PR TITLE
feat(core-flows,utils): add payment + shipment workflow events

### PR DESCRIPTION
**Pull request by riqwan**
what: 

- adds workflow event for payment captured (RESOLVES CORE-2244)
- adds workflow event for payment refunded (RESOLVES CORE-2335)
- adds workflow event for shipment created (RESOLVES CORE-2245)

### CODE REVIEW

File: packages/core/core-flows/src/payment/workflows/capture-payment.ts [hunk 1]
```diff
@@ -13,6 +15,12 @@ export const capturePaymentWorkflow = createWorkflow(
     }>
   ): WorkflowData<PaymentDTO> => {
     const payment = capturePaymentStep(input)
+
```
Comment (carlos-r-l-rodrigues) [inline packages/core/core-flows/src/payment/workflows/capture-payment.ts:18]: Not related to this PR, but as you added the captured event, do you mind to add the refunded as well?

**Review by carlos-r-l-rodrigues — APPROVED** [review]
LGTM

File: packages/core/core-flows/src/payment/workflows/capture-payment.ts (hunk 1 above)
Comment (riqwan) [reply packages/core/core-flows/src/payment/workflows/capture-payment.ts:18]: sure thing 🤞🏻

Comment (riqwan) [reply packages/core/core-flows/src/payment/workflows/capture-payment.ts:18]: Added @carlos-r-l-rodrigues

**Review by olivermrbl — APPROVED** [review]

**Pull request merged**